src/main/java/org/folio/rest/impl/ContactsAPI.java

```diff
@@ -41,31 +41,31 @@ private boolean isInvalidUUID (String errorMessage) {
     return (errorMessage != null && errorMessage.contains("invalid input syntax for uuid"));
   }
 
-  public EdisAPI(Vertx vertx, String tenantId) {
+  public ContactsAPI(Vertx vertx, String tenantId) {
     PostgresClient.getInstance(vertx, tenantId).setIdField(idFieldName);
   }
 
 
   @Override
-  public void getVendorStorageEdis(String query, int offset, int limit, String lang, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+  public void getVendorStorageContacts(String query, int offset, int limit, String lang, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
```

Comment:
I thought we started using PgUtil in mod-vendors... or was that only in mod-orders-storage?  Regardless it should simplify the code a bit and should be considered...

Reply:
Those changes were made as part of the mod-vendors-85 PR, which will be merged in before this PR.  I had forked the off of the mod-vendor-85 branch before you suggested I make those changes.  So once both branches are merged in, the PgUtils will be used for unit tests.

Reply:
that makes this a little confusing and harder to review... so what does that mean for code coverage?

Reply:
Maybe you could rebase off your branch to pull those changes in here...

Reply:
Code rebased and everything looks good